changelog/fix_lint_empty_conditional_body_false_positives.md

```diff
@@ -0,0 +1 @@
+* [#11820](https://github.com/rubocop/rubocop/pull/11820): Fix `Lint/EmptyConditionalBody` false-positives for commented empty `elsif` body. ([@r7kamura][])
```

Comment:
```suggestion
* [#11820](https://github.com/rubocop/rubocop/issues/11820): Fix `Lint/EmptyConditionalBody` false-positives for commented empty `elsif` body. ([@r7kamura][])
```

Reply:
Oh, true, nice catch.
I generated this file using the Rake task like `bundle exec rake changelog:fix[11820]`, but I see that I need an additional step to manually rewrite this if there is the an relevant Issue. Wouldn't it be better to have an option or a separate Rake task  for such cases?